chaosaws/__init__.py

```diff
@@ -68,28 +69,30 @@ def aws_client(resource_name: str, configuration: Configuration = None,
             profile_name=profile_name, region_name=region, **creds)
 
     # default config
-    if configuration.get("aws_role_arn") is None:
+    if aws_assume_role_arn is None or aws_assume_role_arn == "":
+        logger.debug("Using default AWS role")
         return boto3.client(resource_name, region_name=region, **creds)
     else:
+        logger.info("Assuming role: " + aws_assume_role_arn)
```

Comment:
Should this be visible in stdout while running the experiment? We usually try to keep these things light but maybe it's a good idea to be transparent this is running as a different role indeed.

Reply:
My thought was that if you are changing roles, print it. A lot of "errors" I got when changing roles were permission based, where the assumed role didn't have permission to something, so I found it helpful to know I was changing roles and to what role.

Reply:
I think that's fair in this case. We tend to be seldom about the info level so that it concerns itself about telling you the story of the experiment, but I can appreciate offering some context straightaway.